src/layouts/core/Core.component.js

```diff
@@ -1,26 +1,30 @@
 import PropTypes from 'prop-types';
-import React from 'react';
-import { Header, Footer, utils } from '@tidb-community/ui';
+import React, { useContext } from 'react';
+import { Header, Footer, UserProfile, utils } from '@tidb-community/ui';
 import { getData } from '@tidb-community/datasource';
 import { useRouter } from 'next/router';
 
 import { link as linkUtils } from 'utils';
+import { MeContext } from '../../context';
```

Comment:
Please use the absolute path as the mapping rule has been declared in `https://github.com/pingcap/tug-website/blob/CPT-106-CPT-118-FEATURES/jsconfig.json#L3`.

```javascript
import { MeContext } from 'context';
```